tests/OpenCFP/Http/Controller/Admin/TalksControllerTests.php

```diff
@@ -0,0 +1,184 @@
+<?php
+namespace OpenCFP\Tests\Http\Controller\Admin;
+
+use Mockery as m;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use OpenCFP\Application;
+use OpenCFP\Environment;
+
+class AdminTalkControllerTest extends \PHPUnit_Framework_TestCase
+{
+    public $app;
+
+    public function setup()
+    {
+        // Create our Application object
+        $this->app = new Application(BASE_PATH, Environment::testing());
+
+        // Create a test double for our User entity
+        $user = m::mock('OpenCFP\Domain\Entity\User');
```

Comment:
@chartjes 

`README.md` says PHP5.5+ is required, `composer.json` requires `php:>=5.4`, and `.travis.yml` runs builds against at least PHP5.5, so maybe the tests could benefit from using the `class` keyword? Makes it a lot easier to find usages, and also, to refactor. 

What do you think?